content/departments/engineering/dev/admin-exp/devops/managed/index.md

```diff
@@ -15,7 +15,7 @@ For operation guides (e.g. upgrade process), please see [managed instances opera
   - [Monitoring and alerting](#monitoring-and-alerting)
   - [Access](#access)
 - [Cost estimation](cost_estimation.md)
-- [Requesting a managed instance](#requesting-a-managed-instance)
+- [Requesting a managed instance](#workflow)
```

Comment:
Good catch. A little annoying that the CI linter can't detect that these anchor links don't take us anywhere.